### ISSUE TITLE
False positive on UnusedImports with context parameters

### ISSUE THREAD

**Issue by eygraber** [detekt/detekt#8568]
## Expected Behavior

Types used for context parameters should count towards UnusedImports

## Observed Behavior

If a type is imported and is only used for a context parameter, there will be a false positive `UnusedImports` violation

## Steps to Reproduce

```kotlin
import com.example.Foo

context(foo: Foo)
fun doSomethingWithFoo() {
  foo.doSomething()
}

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version of detekt used: 1.23.8
* Version of Gradle used (if applicable): 9.0.0

**Comment by 3flex** [comment]
Can you try a snapshot please?

**Comment by eygraber** [comment]
Not easily because I'm using an external convention plugin that would have to be recompiled with the snapshot as well.

**Comment by eygraber** [comment]
Looks like this is fixed in 2.0.0-alpha.0
